lib/pdf_fill/forms/va210781v2.rb

```diff
@@ -685,19 +713,9 @@ def set_option_indicator
         @form_data['optionIndicator'][selected_option] = true
       end
 
-      def format_other_behavior_details
-        other_behavior = @form_data['otherBehaviors']
-        return if other_behavior.blank?
-
-        details = @form_data['behaviorsDetails']['otherBehavior']
-        @form_data['behaviorsDetails']['otherBehavior'] = "#{other_behavior}: #{details}"
-      end
-
-      def format_police_report_location
-        report = @form_data['reportsDetails']&.[]('police')
-        return if report.blank?
-
-        @form_data['reportsDetails']['police'] = report.values.reject(&:empty?).join(', ')
+      def format_police_details(event)
+        fields = %w[agency city state township country]
+        fields.map { |field| event[field] }.compact.reject(&:empty?).join(', ')
```

Comment:
Did you try using [compact_blank](https://apidock.com/rails/Enumerable/compact_blank) instead of `compact.reject(&:empty?)`? That might work.

Reply:
Power of Rails. I'll attempt the substitution.

Reply:
@rmtolmach Implemented in [89e1e5f](https://github.com/department-of-veterans-affairs/vets-api/pull/20663/commits/89e1e5f348c47cd04402167889da484a4f22fb0f)